{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -212,6 +252,13 @@ def env(name, default=NoDefaultValue, type_=str):
 
 USE_TZ = True
 
+# Media files
+DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
```

Comment:
at some point we should think about a local storage adaptor for the dev environment.